packages/discovery-react-components/src/components/SearchResults/components/Result/Result.tsx

```diff
@@ -113,6 +113,7 @@ export const Result: React.FunctionComponent<ResultProps> = ({
   if (result) {
     documentId = result.document_id;
   }
+
```

Comment:
another option for hover is to just pass the `title=` attribute to the various divs below and the tooltip on hover will show the entire thing (based on your OS/browser)